VCFv4.3.tex

```diff
@@ -406,12 +406,14 @@ \subsubsection{Genotype fields}
       GQ		& 1		& Integer	& Conditional genotype quality \\
       GT		& 1		& String	& Genotype \\
       HQ		& 2		& Integer	& Haplotype quality \\
+      LAA		& . 		& Integer	& Local Alternate Alleles\footnotemark[1]\\ 
+      LPL		& . 		& Integer	& Phred-scaled genotype likelihoods rounded to the closest integer for genotypes that involve the Reference and the LAA alleles only.\footnotemark[1]\\ 
```

Comment:
Shouldn't LPL be sorted after LAD?

Reply:
Both LPL and LAD should be sorted after REF+LAA, the same way as REF+ALT define the order of PL and AD.